distribution/push_v2.go

```diff
@@ -580,14 +600,14 @@ func getMaxMountAndExistenceCheckAttempts(layer PushLayer) (maxMountAttempts, ma
 		// then fallback to upload
 		return 4, 3, true
 
-	// middle sized blobs; if we could not get the size, assume we deal with middle sized blob
+		// middle sized blobs; if we could not get the size, assume we deal with middle sized blob
```

Comment:
This comment should not have the extra indentation.

Reply:
sorry for my ide auto format the code